File: app/src/main/java/im/point/dotty/network/CountersReply.java
package im.point.dotty.network;

import com.google.gson.annotations.SerializedName;

public final class CountersReply extends Envelope {
    int posts;
    int comments;
    @SerializedName("private posts") int privatePosts;
    @SerializedName("private_comments") int privateComments;

    public int getPosts() {
        return posts;
    }

    public int getComments() {
        return comments;
    }

    public int getPrivatePosts() {
        return privatePosts;
    }

    public int getPrivateComments() {
        return privateComments;
    }
}

File: app/src/main/java/im/point/dotty/network/CountersReply.kt
/*
 * Copyright (c) 2019-2021 Pavel A. Sokolov
 */
package im.point.dotty.network

import com.google.gson.annotations.SerializedName

data class CountersReply(var posts: Int? = null,
                         var comments: Int? = null,
                         @SerializedName("private_posts")
                         var privatePosts: Int? = null,
                         @SerializedName("private_comments")
                         var privateComments: Int? = null) : Envelope()